### PR TITLE
Ch-readme 175494047

### PR DESCRIPTION
**Pull request by dianahub250**
### What does this PR do?
This is the creation a readme file which provide all description about VisitAfrica App and it's installation

### Description of Task to be completed?
Reading the readme file and check on all steps for having the project on your local machine and make installations.

### How should this be manually tested?
N/A

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[175494047](https://www.pivotaltracker.com/story/show/175494047)
### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,22 @@ VisitAfrica
 * Rita HATEGEKIMANA
 * Diane NGOGA NIYODUKORERA
```
Comment (mujohn26) [inline README.md]: What is the reason behind of putting these names here.

File: README.md [hunk 2]
```diff
@@ -6,24 +6,22 @@ VisitAfrica
...
 #### Technologies used
 * Flutter
 * Dart
 #### Setup instructions
 * There is no need to enter your personal credentials to access this APP.
-* You can clone this repository to make your own changes by Open terminal by Ctrl+ Alt + T, clone, Type code .to open vs code editor or use android studio. depending on which editor you are using!
+* You can clone this repository to make your own changes by Open terminal by Ctrl+ Alt + T, clone, Type code .to open vs code editor or use android studio. depending on which editor you are using, they can both work!
 * run flutter pub get to install dependencies
 * run flutter run to start
 ####  Contact information
```
Comment (mujohn26) [inline README.md]: -  Change this title to contributors

File: README.md [hunk 3]
```diff
@@ -6,24 +6,22 @@ VisitAfrica
...
 * Email:sumurerwawase620@gmail.com
```
Comment (mujohn26) [inline README.md]: - Please add github profile links for these contributors instead of their emails

**Review by mujohn26 — COMMENTED** [review]
Please apply the listed changes ASAP.

**Review by Sandrine21 — APPROVED** [review]
The description is good. nice job.

**Review by Floride125-art — APPROVED** [review]
I think your readme file is okay. Good job !

File: README.md [hunk 4]
```diff
@@ -1,29 +1,23 @@
...
 * run flutter run to start
-####  Contact information
-* Email:sumurerwawase620@gmail.com
-* Email:ftuyisenge120@gmail.com
-* Email:rhategekimana920@gmail.com
-* Email:dniyodukorera120@gmail.com
-#### Link to our Live pages 
-
+####  Contributers
+* Sandrine21
```
Comment (mujohn26) [inline README.md]: Please Diane I said to add Github link of the contributors not their github usernames.

**Review by mujohn26 — APPROVED** [review]
Good job! but always remember to squash your commits.

**Pull request merged**